src/RemixDownloader/RemixDownloader.Console/Program.cs

```diff
@@ -54,9 +57,7 @@ public static async Task Main(string[] args)
                 includeOptimizedString = "n";
             }
 
-            bool includeOptimized = includeOptimizedString == "y" || 
-                                    includeOptimizedString == "e" || 
-                                    includeOptimizedString == "s";
```

Comment:
Wasn't sure about this part -- seems that only checking for `y` (and maybe `Y`?) should be enough?

Reply:
This was to appease some folks who were used to other CLIs that let you use any letter from `yes`. If it becomes an issue on capitalization, I can add a `ToLower()` to flatten everything out before checking.